examples/apps/cra-demo/src/App.js

```diff
@@ -0,0 +1,70 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import React from "react";
+import TinyliciousClient from "@fluid-experimental/tinylicious-client";
+import { SharedMap } from "@fluidframework/map";
+
+const getContainerId = () => {
+    let isNew = false;
+    if (window.location.hash.length === 0) {
+        isNew = true;
+        window.location.hash = Date.now().toString();
+    }
+    const id = window.location.hash.substring(1);
+    return { id, isNew };
+};
+
+TinyliciousClient.init();
+
+const initFluidData = async (setData) => {
+    const containerConfig = {
```

Comment:
naming: containerSchema?
```suggestion
    const containerSchema = {
```